_data/wai-evaluation-tools-list/submissions/a401fe50-04d3-11f0-9d9e-697970187972.json

```diff
@@ -0,0 +1,60 @@
+{
+  "a11yloc": "",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results",
+    "Displaying information within web pages",
+    "Creating an accessibility score",
+    "Providing step-by-step evaluation guidance"
+  ],
+  "automated": [
+    "Component",
+    "Single page/screen",
+    "Sample of pages/screens",
+    "Entire website / software / product",
+    "Restricted or password protected pages"
+  ],
+  "comments": "",
+  "contact": "karthik@browserstack.com",
+  "desktop": [
+    "Windows",
+    "MacOS"
+  ],
+  "features": "Run accessibility checks within your test automation pipeline with just one line of code—compatible with any framework & no maintenance required. DOM updates are tracked automatically. Get detailed reports, track trends, and catch accessibility issues early.",
+  "form_name": "submission",
+  "form_version": "1",
+  "guideline": [
+    "WCAG 2.2",
+    "WCAG 2.1",
+    "WCAG 2.0"
```

Comment:
@josehilera I am looking at the [documentation](https://www.browserstack.com/docs/accessibility/automated-tests/configuration-options): would you ask the vendor if `wcag2a`, `wcag2aa` and `wcag2aaa` relate to WCAG 2.0 or to the "latest version of WCAG"? Or more broadly, if the user can target WCAG 2.0?